benches/event.rs

```diff
@@ -92,17 +81,6 @@ fn benchmark_event_create(c: &mut Criterion) {
     });
 }
 
-fn create_event(json: Value) -> LogEvent {
```

Comment:
What a weird way to create an event for benchmarking 🤔

Reply:
Yeah. 🤷🏻